packages/@react-native-windows/cli/src/generator-windows/index.ts

```diff
@@ -95,6 +95,34 @@ export async function copyProjectTemplateAndReplace(
     namespace = namespace.split(/[.:]+/).map(pascalCase).join('.');
   }
 
+  // Checking if we're overwriting an existing project and re-uses their projectGUID
+  const projectPathVcx = path.join(
```

Comment:
Don't forget the root `destPath`. Also, rather  than creating two path variables, this could be simplified to use the language option passed by the caller, to something like:

```js
const existingProjectPath = path.join(destPath, windowsDir, newProjectName, newProjectName + (language === 'cs' ? '.csproj' : '.vcxproj'));
```

Reply:
oh good call out, I totally forgot about the language option! :)